src/components/EventBanner/EventBanner.tsx

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import "./EventBanner.scss";
+import dayjs from "dayjs";
+import * as SEC from "../SEC/lib.js";
+
+interface EventBannerProps {
+	flyerSource: any;
+	altText: string;
+	title: string;
+	desc: string;
+	startDate: dayjs.Dayjs;
+	endDate: dayjs.Dayjs;
+	meetingLink: string;
```

Comment:
definitely make this optional
```suggestion
	meetingLink?: string;
```